interview-questions.md

```diff
@@ -8,50 +8,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a genderic input for a function while the arguement is what the function will be replacing the generic value in the parameter
 
-Researched answer:
+Researched answer: After researching, an arguement is a value declared within the function itself while a parameter is the variables that are defined when a function is declared.  So for example 
+
+```Javascript
+const myFunc = (num1, num2) =>{
+    return num1 + num2
+}
+console.log(myFunc(3, 4))
+```
+The parameters are the values num1 and num 2 because they are being defined when the function is declared.  Where as the numbers 3 and 4 are the arguments because they are being declared inside myFunc() function.
```

Comment:
great example. For clarification, the argument is the value used during the function call aka function invocation aka invoking the function.